Common/Sources/Helper/ThemeManager.swift

```diff
@@ -46,7 +46,7 @@ public class ThemeManager: NSObject {
             proxy.largeTitleTextAttributes =
                 [
                     NSAttributedString.Key.foregroundColor: Theme.NavigationBar.largeTitleColor,
-                    NSAttributedString.Key.font: Theme.NavigationBar.largeTitleFont
+                    NSAttributedString.Key.font: Theme.NavigationBar.titleFont
```

Comment:
what's happening if you use largeTitleFont instead of titleFont because you have decreased that to 20 as well

Reply:
Code updated for SE 1st Gen